lib/files_test.go

```diff
@@ -233,11 +233,11 @@ func TestWriteLines(t *testing.T) {
 			}
 		}
 
+		file.Close()
```

Comment:
Everything looks good. My only question here is why not use `defer` to close the file?

Reply:
cheers warren, i came across a race condition between `file.Close()` and https://github.com/warrensbox/terraform-switcher/blob/309e0ba04a08c66b3a17a06cf1b0b86a51f9c1e8/lib/files_test.go#L240,

i believe the os is trying to remove the `installLocation`, however it couldn't do so until the `file.Close()` is complete